java/java.source/src/org/netbeans/modules/java/source/package-info.java

```diff
@@ -20,10 +20,11 @@
 @TemplateRegistrations({
     @TemplateRegistration(folder = "Classes/Code", position = 100, content = "resources/GeneratedMethodBody.template", scriptEngine = "freemarker", displayName = "#GeneratedMethodBody", category = "java-code-snippet"),
     @TemplateRegistration(folder = "Classes/Code", position = 200, content = "resources/OverriddenMethodBody.template", scriptEngine = "freemarker", displayName = "#OverriddenMethodBody", category = "java-code-snippet"),
-    @TemplateRegistration(folder = "Classes/Code", position = 300, content = "resources/LambdaBody.template", scriptEngine = "freemarker", displayName = "#LambdaBody", category = "java-code-snippet")
+    @TemplateRegistration(folder = "Classes/Code", position = 300, content = "resources/LambdaBody.template", scriptEngine = "freemarker", displayName = "#LambdaBody", category = "java-code-snippet"),
+    @TemplateRegistration(folder = "Classes/Code", position = 400, content = "resources/LambdaExpression.template", scriptEngine = "freemarker", displayName = "#LambdaExpression", category = "java-code-snippet")
 })
 @NbBundle.Messages({
-    "GeneratedMethodBody=Generated Method Body", "OverriddenMethodBody=Overridden Method Body", "LambdaBody=Generated Lambda Body" //NOI18N
+    "GeneratedMethodBody=Generated Method Body", "OverriddenMethodBody=Overridden Method Body", "LambdaBody=Generated Lambda Body", "LambdaExpression=Generated Lambda Expression" //NOI18N
 })
```

Comment:
@dbalek is it possible to hide the Code Snippets for lambdas somehow so that the user can't edit them via the template window?

Reply:
@jtulach do you happen to know if that would be possible? To hide the snipped from the user?

Reply:
If you don't want the user to edit the template, why put in a user accessible position in the first place? can't the template be read from the classpath or defined inline?